[2025-08-14T17:20:00.514+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-08-14T17:20:00.540+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: orders_pipeline.clear_output_prefix manual__2025-08-14T17:19:36.646942+00:00 [queued]>
[2025-08-14T17:20:00.551+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: orders_pipeline.clear_output_prefix manual__2025-08-14T17:19:36.646942+00:00 [queued]>
[2025-08-14T17:20:00.551+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-08-14T17:20:00.567+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): clear_output_prefix> on 2025-08-14 17:19:36.646942+00:00
[2025-08-14T17:20:00.573+0000] {standard_task_runner.py:64} INFO - Started process 62 to run task
[2025-08-14T17:20:00.577+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'orders_pipeline', 'clear_output_prefix', 'manual__2025-08-14T17:19:36.646942+00:00', '--job-id', '18', '--raw', '--subdir', 'DAGS_FOLDER/orders_pipeline.py', '--cfg-path', '/tmp/tmpqkrrngf2']
[2025-08-14T17:20:00.579+0000] {standard_task_runner.py:91} INFO - Job 18: Subtask clear_output_prefix
[2025-08-14T17:20:00.661+0000] {task_command.py:426} INFO - Running <TaskInstance: orders_pipeline.clear_output_prefix manual__2025-08-14T17:19:36.646942+00:00 [running]> on host acbcfe9b9346
[2025-08-14T17:20:00.815+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='data-eng' AIRFLOW_CTX_DAG_ID='orders_pipeline' AIRFLOW_CTX_TASK_ID='clear_output_prefix' AIRFLOW_CTX_EXECUTION_DATE='2025-08-14T17:19:36.646942+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-08-14T17:19:36.646942+00:00'
[2025-08-14T17:20:00.817+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-08-14T17:20:00.876+0000] {credentials.py:1147} INFO - Found credentials in environment variables.
[2025-08-14T17:20:02.106+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-08-14T17:20:02.107+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-08-14T17:20:02.130+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=orders_pipeline, task_id=clear_output_prefix, run_id=manual__2025-08-14T17:19:36.646942+00:00, execution_date=20250814T171936, start_date=20250814T172000, end_date=20250814T172002
[2025-08-14T17:20:02.198+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2025-08-14T17:20:02.237+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-08-14T17:20:02.240+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
